impl/src/settings.rs

```diff
@@ -531,6 +531,45 @@ impl RawRazeSettings {
       || self.rust_rules_workspace_name.is_some()
       || self.vendor_dir.is_some()
   }
+
+  fn print_deprecation_warnings(&self) {
+    if self.incompatible_relative_workspace_path.is_some() {
+      eprintln!(
+        "WARNING: `[*.raze.incompatible_relative_workspace_path]` will soon be deprecated. Please \
+         update your project to use the `true` behavior."
+      );
+    }
+
+    if self.target.is_some() {
+      eprintln!(
+        "WARNING: `[*.raze.target]` will soon be deprecated. Please update your project to use \
+         `[*.raze.targets]`."
+      );
+    }
+
+    if self.default_gen_buildrs.is_none() {
+      eprintln!(
+        "WARNING: The default of `[*.raze.default_gen_buildrs]` will soon be set to `true`. \
+         Please explicitly set this flag to prevent a change in behavior."
+      );
+    }
+
+    if self.rust_rules_workspace_name.is_none() {
+      eprintln!(
+        "WARNING: The default of `[*.raze.rust_rules_workspace_name]` will soon be set to \
```

Comment:
Should we include the old value of the workspace name here? Would help folks search their code to know whether or not they're affected.

Reply:
Updated